cmd/kpod/ps.go

```diff
@@ -103,6 +122,10 @@ var (
 			Name:  "size, s",
 			Usage: "Display the total file sizes",
 		},
+		cli.BoolFlag{
+			Name:  "verbose, v",
```

Comment:
I don't like verbose, I would prefer to be --ns or --namespace

Reply:
fixed